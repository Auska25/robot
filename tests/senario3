PLACE 2,2,NORTH
REPORT
MOVE
REPORT
MOVE
REPORT
MOVE
REPORT
LEFT
REPORT
MOVE
MOVE
MOVE
REPORT
RIGHT
RIGHT
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT
LEFT
LEFT
LEFT
LEFT
REPORT
RIGHT
REPORT
MOVE
MOVE
MOVE
MOVE
MOVE
REPORT
QUIT
