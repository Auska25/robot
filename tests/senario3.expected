SYSTEM START
2,2,NORTH
2,3,NORTH
2,4,NORTH
2,4,NORTH
2,4,WEST
0,4,WEST
4,4,EAST
4,4,EAST
4,4,SOUTH
4,0,SOUTH
SYSTEM EXIT
